src/main/scala/tilelink/Parameters.scala

```diff
@@ -534,48 +537,81 @@ class TLSlavePortParameters private(
   // Does this Port manage this ID/address?
   def containsSafe(address: UInt) = findSafe(address).reduce(_ || _)
 
-  private def supportHelper(
+  private def addressHelper(
       safe:    Boolean,
-      member:  TLManagerParameters => TransferSizes,
+      member:  TLSlaveParameters => TransferSizes,
       address: UInt,
       lgSize:  UInt,
-      range:   Option[TransferSizes]): Bool = {
+      range:   Option[TransferSizes]): Bool = { // What does the range argument do?
```

Comment:
`// range provides a limit on the sizes that are expected to be sent, which might reduce circuit complexity`